lib/src/data/model/hearthstone_card.dart

```diff
@@ -31,6 +31,26 @@ class HearthstoneCard {
     this.image,
   });
 
+  Map<String, dynamic> toMap() {
+    final Map<String, dynamic> cardMap = {};
+
+    cardMap['idCard'] = cardId;
+    cardMap['cardSet'] = cardSet;
+    cardMap['playerClass'] = playerClass;
+    cardMap['name'] = name;
+    cardMap['type'] = type;
+    cardMap['text'] = text;
+    cardMap['health'] = health;
+    cardMap['cost'] = cost;
+    cardMap['attack'] = attack;
+    cardMap['faction'] = faction;
+    cardMap['locale'] = locale;
+    cardMap['mechanics'] = mechanics;
+    cardMap['image'] = image;
+
+    return cardMap;
+  }
```

Comment:
Instead to do this, you can return directly a map<string, dynamic>
return {
     'idCard': cardId,
     'cardSet': cardSet,
     .....
}

Reply:
I made that method when I was trying to incorporate Firestore to the app and forgot to delete it, I found a better, more direct way to do it so I deleted the method.